ui/src/plot.tsx

```diff
@@ -735,6 +735,11 @@ const
       flexGrow: 1,
       display: 'flex',
     },
+    plotContainer: {
+      $nest: {
+        '>div': { textAlign: 'center' }
```

Comment:
This does not fix the problem, which is that the G2 vis for some reason does not use all available width. This change simply centers the plot.